src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.db;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.Transaction;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class Storage {
+    private final Map<Fruit, List<Transaction>> transactionsData;
```

Comment:
I don't like idea to have in storage list of transactions.
It's not intuitive, when you come into real life storage you expect to see some amount of some product stored there